target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -76,6 +76,12 @@ ath79_setup_interfaces()
 	yuncore,a770)
 		ucidef_set_interfaces_lan_wan "eth1" "eth0"
 		;;
+        comfast,cf-wr650ac-v1|\
+        comfast,cf-wr650ac-v2)
```

Comment:
These two lines still have leading spaces.
Despite, you are changing the execute bit on this file. (I do not think it should be 755 for board.d files, though. I will send separate patch to the list for that).